sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ProtoDescriptorGenerator.scala

```diff
@@ -46,6 +49,10 @@ object ProtoDescriptorGenerator {
     messages.foreach(protoBuilder.addMessageType)
 
     // finally build all final descriptor
-    Descriptors.FileDescriptor.buildFrom(protoBuilder.build, dependencies)
+    val fd = Descriptors.FileDescriptor.buildFrom(protoBuilder.build, dependencies)
+    if (logger.isDebugEnabled) {
+      logger.debug("Generated file descriptor for service [{}]: {}", name, ProtoDescriptorRenderer.toString(fd))
```

Comment:
This render is doing a great job, but this log puts the first line of the descriptor together with the main logging message. After that the rest appears beautifully formatted. 

I'm sure you did it to drive me crazy. It was fun, but can we add a break line now? 😆 

```suggestion
      logger.debug("Generated file descriptor for service [{}]: \n{}", name, ProtoDescriptorRenderer.toString(fd))
```